examples/step-44/step-44.cc

```diff
@@ -2391,194 +2391,183 @@ namespace Step44
 
   // @sect4{Solid::make_constraints}
   // The constraints for this problem are simple to describe.
-  // However, since we are dealing with an iterative Newton method,
-  // it should be noted that any displacement constraints should only
-  // be specified at the zeroth iteration and subsequently no
-  // additional contributions are to be made since the constraints
-  // are already exactly satisfied.
+  // In this particular example, the boundary values will be calculated for
+  // the two first iterations of Newton's algorithm. In general, one would
+  // build non-homogeneous constraints in the zeroth iteration (that is, when
+  // `apply_dirichlet_bc == true` in the code block that follows) and build
+  // only the corresponding homogeneous constraints in the following step. While
+  // the current example has only homogeneous constraints, previous experiences
+  // have shown that a common error is forgetting to add the extra condition
+  // when refactoring the code to specific uses. This could lead errors that are
```

Comment:
I think there is a typo?
```suggestion
  // when refactoring the code to specific uses. This could lead to errors that are
```

Reply:
Thanks for the review, and for spotting this mistake! I've corrected it now.